pkg/utils/crtl_utils.go

```diff
@@ -86,6 +86,7 @@ func ContainsString(slice []string, s string) bool {
 	return false
 }
 
+//Remove strings in a  array which equals to the given string.
```

Comment:
You should use the func name as the initial word of the comment. Please change it. Thanks.